test/fixtures-import.js

```diff
@@ -136,3 +136,5 @@ verifyIncludeRawText(
 )
 
 Promise.all(promised)
+
+Promise.all(promised)
```

Comment:
@DerDrodt Just making sure - this line is there twice on purpose, right?

Reply:
That seems to be merge error. Fixed.